extensions/PhabBugz/lib/API/V1/Lando.pm

```diff
@@ -0,0 +1,131 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::Extension::PhabBugz::API::V1::Lando;
+
+use 5.10.1;
+use Mojo::Base qw( Mojolicious::Controller );
+
+use JSON::Validator::Joi 'joi';
+use Mojo::JSON qw(true false);
+
+use Bugzilla::Constants;
+use Bugzilla::Group;
+
+use Bugzilla::Extension::PhabBugz::Constants;
+use Bugzilla::Extension::TrackingFlags::Flag;
+
+our %api_field_names = reverse %{Bugzilla::Bug::FIELD_MAP()};
+$api_field_names{'bug_group'} = 'groups';
+
+sub setup_routes {
+  my ($class, $r) = @_;
+  $r->get('/lando/uplift/:id')->to('PhabBugz::API::V1::Lando#get');
+  $r->put('/lando/uplift')->to('PhabBugz::API::V1::Lando#uplift');
+}
+
+sub get {
+  my ($self) = @_;
+  Bugzilla->usage_mode(USAGE_MODE_MOJO_REST);
+
+  my $user = $self->bugzilla->login;
+  $user->id || return $self->user_error('login_required');
+
+  # Must be lando automation user to access this endpoint
+  ($user->login eq LANDO_AUTOMATION_USER)
+    || return $self->user_error('login_required');
+
+  # Upgrade Lando user permissions to make changes to any bug
+  $user->{groups}       = [Bugzilla::Group->get_all];
+  $user->{bless_groups} = [Bugzilla::Group->get_all];
+
+  my $id = $self->param('id');
+  defined $id || ThrowCodeError('param_required', {param => 'id'});
+
+  my $bug = Bugzilla::Bug->check($id);
```

Comment:
The `/rest/bug` endpoint takes an `id` param where the `id` is a comma-separated list of bug IDs. Can we use that logic here as well?